src/website/app/demos/Card/examples/card-block/arbitraryChildren.js

```diff
@@ -30,8 +30,9 @@ export default {
   title: 'Arbitrary children',
   // $FlowFixMe
   backgroundColor: mineralTheme.color_gray_10,
-  description:
-    'A CardBlock will render any children. For best results, please make sure your content has no top/bottom margin or left/right padding.',
+  description: `A Card will render any children.
+
+For best results, please ensure content matches the top/bottom margin and left/right padding of the other \`<Card*>\` components, or use [CardBlock]('./card-block').`,
   scope: { Button, Card, CardBlock, CardTitle, CustomContent, DemoLayout },
   source: `
     <DemoLayout>
```

Comment:
I can't comment on the direct line (40), but it should be "Card Title" to match the convention used elsewhere.